### DISCUSSION TITLE
Show from underneath AppBar

### DISCUSSION THREAD

**Discussion by doughboyks** [cb-cloud/flutter_in_app_notification#24] [Ideas]
I love this.  The only thing I wished is that I could get it to appear as if it was a growl from beneath the appbar. I added a margin so it would stop underneath it, but it floats down over it.  Do you have any suggestions?

**Comment by Kurogoma4D** [comment]
Hi @doughboyks .
I thought it would be able to put `InAppNotification` widget into your Scaffold body.
But it maybe doesn't work because `screenSize` state is obtained from `MediaQuery`, so your usage needs some internal modification.

Could you make PR? Or I'll get to it, but probably need some time. Thanks.
